### PR TITLE
Feature/7 basic logging interface

### LINKED ISSUE #7: Basic Logging interface
## User stories
As a developer I want to have clear defined logging interface in order to know what I can and have to log

## (Wiki) References
- [Logging](https://github.com/BP2022-AP1/bp2022-ap1/wiki/Logging)

## Acceptance Criteria
- [x] Create Logger Class
- [x] Create Logger function endpoints
- [x] document when to use functions in Wiki and Code with usage in wiki and code

### PR DESCRIPTION
**Pull request by LucasDerReisende**
Closes #7 

This PR includes the Logger class as well as all the basic interfaces, so they can be used by others.
Please keep in mind, that this PR **depends** on #23!

## PR checklist

- [ ] Acceptance criteria fulfilled
- [ ] Additional features are tested
- [ ] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: tests/test_sample.py
Comment (CR1337) [inline tests/test_sample.py:1]: How does this contribute to the user story? Maybe delete this?

File: src/logger/logger.py
Comment (CR1337) [inline src/logger/logger.py:1]: For empty methods there are conventions in Python.

A method that will get implemented with a later ticket:
`
def method():
    raise NotImplementedError("method is not implemented yet (or some other message)")
`

A method that is intentionally empty and will stay empty:
`
def method():
    pass
`

An abstract method:
`
@abstractmethod
def method():
    raise NotImplementedError("message")
`

Comment (CR1337) [reply src/logger/logger.py:1]: There should be newlines in the code segments. I guess GitHub Markdown messed it up.

**Review by CR1337 — CHANGES_REQUESTED** [review]
See comments on the files.

File: src/logger/logger.py
Comment (LucasDerReisende) [reply src/logger/logger.py:1]: Should be changed now

**Review by CR1337 — APPROVED** [review]
look good! I like.

**Pull request merged**